hcloud/load_balancers.go

```diff
@@ -87,6 +91,57 @@ func (l *loadBalancers) GetLoadBalancerName(ctx context.Context, clusterName str
 	return cloudprovider.DefaultLoadBalancerName(service)
 }
 
+// MustGetKey creates a key from Kubernetes resource in the format <namespace>/<name>
+func MustGetKey(obj interface{}) string {
+	if obj == nil {
+		return ""
+	}
+
+	o := MustGetObject(obj)
+	return fmt.Sprintf("%s/%s", o.GetNamespace(), o.GetName())
```

Comment:
If `MustGetObject` returns nil this may panic if whatever implements `metav1.Object` is not nil safe. You need to check if `o` is `nil`.